airflow/contrib/operators/sqoop_operator.py

```diff
@@ -168,16 +194,18 @@ def execute(self, context):
                     split_by=self.split_by,
                     where=self.where,
                     direct=self.direct,
-                    driver=self.driver)
-            elif not self.query:
+                    driver=self.driver,
+                    extra_import_options=self.extra_import_options)
+            elif self.query:
                 hook.import_query(
-                    query=self.table,
+                    query=self.query,
```

Comment:
this line is clearly wrong. Shouldn't be `query=self.table` but rather `query=self.query`

Reply:
👍